scripts/common/docker.sh

```diff
@@ -35,10 +35,6 @@ function install_docker() {
 
     if [ -z "$PRESERVE_DOCKER_CONFIG" ] && [ -n "$PROXY_ADDRESS" ]; then
         docker_configure_proxy
-        local dockerProxy=$(docker info 2>/dev/null | grep -i "HTTP Proxy:")
-        if ! echo "$dockerProxy" | grep -q "$PROXY_ADDRESS"; then
```

Comment:
This fails with an authenticating proxy because docker info redacts the credentials in the URL.